routes/mentors.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+
+const router = express.Router();
+const mentorService = require("../services/mentor_service");
+const { ROLE_NAMES } = require("../utils");
+const {verifyRole} = require("../middleware/authorization");
+const { isValidIntegerId, respondWithError } = require("./helpers");
+const logger = require('../logger')(__filename);
+
+router.use(express.json());
+ 
+// Get all mentors
+router.get("/mentors", async (req, res) => {
+  try {
+    const { status } = req.query;
+    const mentors = await mentorService.getMentors(req.userId, status);
+    res.json({ mentors });
+  } catch (err) {
+    if (err.message.includes('not permitted')) {
+      logger.warn(`Unauthorized mentor list request: ${err.message}`);
+      return respondWithError(res, 403, "You don't have permission to view these mentors");
```

Comment:
I think we're safe to pass error.message in the response as well. 
this might help frontend understand what's going on as well & maybe produce a better output for the user